main.go

```diff
@@ -348,61 +348,19 @@ func renderHandler(w http.ResponseWriter, r *http.Request, stats *renderStats) {
 				continue
 			}
 
-			var glob pb.GlobResponse
-			var haveCacheData bool
-
-			if response, ok := findCache.get(m.Metric); useCache && ok {
```

Comment:
This removes the only use of `findCache`.  Please clean up the rest of the initialization for this variable.